oeis.py

```diff
@@ -169,6 +169,22 @@ def A000040(start: int = 0, limit: int = 20) -> Collection[int]:
     return result
 
 
+@oeis
+def A023811(start, limit):
```

Comment:
`start` and `limit` should have a default value, and like any other implementations in this file, type annotations, like:
```suggestion
def A023811(start: int = 0, limit: int = 20) -> Collection[int]:
```